plugins/inventory/foreman.py

```diff
@@ -85,6 +85,64 @@
         description: Number of hosts per batch that will be retrieved from the Foreman API per individual call
         type: integer
         default: 250
+      foreman:
+        description: foreman server related configuration
+        type: dict
+        use_reports_api:
+            description: Use Reporting API.
+            type: boolean
```

Comment:
thes should be `type: bool`, not that Ansible properly parses sub-options for inventories anyways.

Reply:
I see type: boolean in other params in the file. Same with type: integer ..

Reply:
yeah, not sure, maybe it's not strict